overlord/snapstate/snapstate.go

```diff
@@ -314,6 +314,27 @@ func refreshRetain(st *state.State) int {
 	return retain
 }
 
+// maxInhibitionTime returns the value of the maximum inhibition time, in seconds
```

Comment:
The comment probably needs to be updated since the return is a time.Duration, not the number of seconds

Reply:
Good point!